docs/minima_tracer.rst

```diff
@@ -0,0 +1,13 @@
+.. _new_page:
+
+Minima Tracer
+==============
+
+.. doxygenclass:: BSMPT::MinimumTracer
+   :project: BSMPT
+   :members:
+   :protected-members:
+   :private-members:
```

Comment:
```suggestion
```

I think we can remove the private members.
They are only of interest for the internal development